terraform/terraform_spec.rb

```diff
@@ -23,6 +23,14 @@
     expect(file("/etc/alpine-release")).to be_file
   end
 
+  it "installs the right version of Alpine" do
+    expect(os_version).to include("Alpine Linux 3.3")
+  end
```

Comment:
I feel more comfortable pinning the versions of dependencies so upgrading terraform version is a minimal change, instead or risking unexpected behaviour due to dependencies.